ocs_ci/ocs/resources/machineconfig.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf8 -*-
+
+
+import logging
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
+
+
+logger = logging.getLogger(__name__)
+
+
+def deploy_machineconfig(tmp_path, mc_name, mc_dict, mcp_num=2):
+    """
+    Deploy given MachineConfig dict and wait for the configuration to be
+    deployed on all MachineConfigPools.
+    """
```

Comment:
Please describe the arguments of this function in doc string.

Reply:
done